defaults/main.yml

```diff
@@ -14,7 +14,7 @@ sensu_enterprise_dashboard_package: sensu-enterprise-dashboard
 # Sensu repo urls
 sensu_yum_repo_url: "http://repositories.sensuapp.org/yum/$releasever/$basearch/"
 sensu_apt_repo_url: "deb     http://repositories.sensuapp.org/apt {{ ansible_distribution_release }} main"
-sensu_apt_key_url: "http://repositories.sensuapp.org/apt/pubkey.gpg"
+sensu_apt_key_url: "https://repositories.sensuapp.org/apt/pubkey.gpg"
```

Comment:
Is this supposed to be changed to `https://sensu.global.ssl.fastly.net/` instead?

Reply:
I think both are useable, but https://sensu.global.ssl.fastly.net/ matches the documentation (i.e. https://sensuapp.org/docs/latest/platforms/sensu-on-ubuntu-debian.html), so it's the better choice. This would apply to the other 2 PRs as well.